data/json/items/armor/helmets.json

```
@@ -586,8 +586,15 @@
     "encumbrance": 23,
     "warmth": 30,
     "material_thickness": 8,
+    "max_charges": 100,
+    "initial_charges": 100,
+    "charges_per_use": 1,
+    "ammo": "gasfilter_m",
+    "use_action": "GASMASK",
     "environmental_protection": 7,
+    "environmental_protection_with_filter": 16,
```

While gas mask has 16 EP, all the survivor masks only have 15, and they're also modified gas masks so it might be reasonable to assume modifying a gas mask into this has the same effect.
```suggestion
    "environmental_protection": 1,
    "environmental_protection_with_filter": 15,
```